src/crab/ebpf_domain.hpp

```diff
@@ -1120,6 +1137,22 @@ class ebpf_domain_t final {
         return o;
     }
 
+    static void initalize_packet(ebpf_domain_t& inv) {
+        using namespace dsl_syntax;
+
+        inv -= variable_t::packet_size();
+        inv -= variable_t::meta_offset();
+
+        inv += 0 <= variable_t::packet_size();
+        inv += variable_t::packet_size() < MAX_PACKET_OFF;
+        if (global_program_info.type.context_descriptor->meta >= 0) {
+            inv += variable_t::meta_offset() <= 0;
+            inv += variable_t::meta_offset() >= -4098;
```

Comment:
A comment would be helpful to explain where the number -4098 came from.
(Yes I know the line was just moved from old line 1143 so this is not a comment on this change per se.)

Reply:
I think I made this number up, but I'm not sure.

Reply:
I think this number should be taken from `ebpf_platform`.